.github/workflows/build_and_run_pre_encrypted_ota.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and Run Pre Encrypted OTA Example
```

Comment:
All examples in this repository will be run after this PR https://github.com/espressif/idf-extra-components/pull/183 is merged. Could you wait & rebase after it is merged?